app/components/show/apo/overview_component.rb

```diff
@@ -7,6 +7,11 @@ class OverviewComponent < ApplicationComponent
       def initialize(presenter:)
         @presenter = presenter
         @solr_document = presenter.document
+        @registration_workflow = @presenter.cocina.administrative.registrationWorkflow
```

Comment:
Very minor: could this line use `presenter` (the kwarg) instead of `@presenter` (the instance variable)? There's no functional difference, given the code on line 8... my thinking is that the kwarg might be preferable on line 10 because that's how we're referring to the presenter on the prior line. (Again... very minor!)

Reply:
Makes sense to me, and it works, of course. Pushing that update.